### ISSUE TITLE
RadListView crashes on release snapshot build with NoClassDefFoundError for SimpleItemAnimator

### ISSUE THREAD

**Issue by PeterStaev** [telerik/nativescript-ui-feedback#360]
I have a RadListView that uses swipe actions. On debug builds everything works as expected. But with a `--release --snaphsot` the view crashes. This is a non-angular app. Below is a screenshot from the crash. 
![simpleanimator](https://user-images.githubusercontent.com/9464097/31985247-24728cb8-b96d-11e7-8dc1-6b1f8c9c9980.jpg)

**Comment by PeterStaev** [comment]
This was because the manually added compile dependency was set to version 21.0.+ which is wrong as the use `SimpleItemAnimator` class was introduced in version 23.0.1. Adding a higher version fixed the crash. I will leave this open just to make sure the fix for the manually added compile dependencies includes the correct support lib version.

**Comment by VladimirAmiorkov** [comment]
Hi @PeterStaev ,

Yes you are correct. We did have an issue with a missing include.grade in the nativescript-pro-ui@3.1.0 but this has been resolved in the latest **3.2.0**. Just to be clear here is what the current version of the plugin uses as its native Android dependencies:

```
dependencies {
	compile "com.android.support:appcompat-v7:23.1.0"
	compile "com.android.support:recyclerview-v7:23.1.0"
}
```

I will close this issue.
